components/automate-ui/src/app/page-components/navbar/navbar.component.html

```diff
@@ -5,28 +5,49 @@
         <a tabindex="0" routerLink="/" class="logo" title="logo link to homepage"></a>
       </div>
       <div class="navigation-menu">
-        <div role="menuitem" class="nav-link"><a routerLink="/dashboards/event-feed" routerLinkActive="active" tabindex="0">Dashboards</a></div>
+        <app-authorized [allOf]="[
+            ['/api/v0/eventfeed', 'get'],
+            ['/api/v0/eventstrings', 'get'],
+            ['/api/v0/event_type_counts', 'get'],
+            ['/api/v0/event_task_counts', 'get'],
+            ['/api/v0/cfgmgmt/suggestions', 'get']]">
+          <div role="menuitem" class="nav-link"><a routerLink="/dashboards/event-feed" routerLinkActive="active" tabindex="0">Dashboards</a></div>
```

Comment:
The only substantive change in this file is adding `<app-authorized>` to this element.
All the remaining changes are reformatting one-per-line for clarity.